lib/index.js

```diff
@@ -52,6 +52,9 @@ function exec(command, ...args) {
   return deferred.promise;
 }
 
+const endsWith = (suffix, str) =>
+  str.indexOf(suffix, str.length - suffix.length) !== -1;
+
```

Comment:
It's because of the `selfContained` option? Anyway, I like not to pollute global namespace.